readme.md

```diff
@@ -1101,4 +1101,4 @@ Please see our [contributing.md](./contributing.md)
 - Naoyuki Kanezawa ([@nkzawa](https://twitter.com/nkzawa)) – ▲ZEIT
 - Tony Kovanen ([@tonykovanen](https://twitter.com/tonykovanen)) – ▲ZEIT
 - Guillermo Rauch ([@rauchg](https://twitter.com/rauchg)) – ▲ZEIT
-- Dan Zajdband ([@impronunciable](https://twitter.com/impronunciable)) – Knight-Mozilla / Coral Project
\ No newline at end of file
+- Dan Zajdband ([@impronunciable](https://twitter.com/impronunciable)) – Knight-Mozilla / Coral Project
```

Comment:
Hmm, I just edited this on GitHub - looks like that editor is adding a trailing new line. I can do this manually and strip it if you want.